nala/utils/global.setup.js

```diff
@@ -29,7 +29,7 @@ async function getGitHubPRBranchLiveUrl() {
   const prFromOrg = process.env.prOrg || toRepoOrg;
   const prFromRepoName = process.env.prRepo || toRepoName;
 
-  const prBranchLiveUrl = `https://${prBranch}--${prFromRepoName}--${prFromOrg}.hlx.live`;
+  const prBranchLiveUrl = `https://${prBranch}--${prFromRepoName}--${prFromOrg}.aem.live`;
```

Comment:
Could the repo be something else other than Milo? This might cause issues, since most consumers are not using `.aem.`

Reply:
I was wondering about it, but given that it works 🤔 
@skumar09 is this exported and used by consumers, or if the nala tests pass on milo, we are good?